src/ode_solve.jl

```diff
@@ -146,7 +146,7 @@ function ode_dfdx end
 function ode_dfdx(phi::ODEPhi{C, T, U}, t::Number, θ,
                   autodiff::Bool) where {C, T, U <: Number}
     if autodiff
-        ForwardDiff.derivative(t -> phi(t, θ), t)
+        Diffractor.PrimeDerivativeFwd(t -> phi(t, θ))(t)
```

Comment:
can this use reverse diff? Diffractor reversediff is in much better shape.

Reply:
Given the sizes, Forward mode makes more sense here.